src/internal/utils/__tests__/global-flags-ssr.test.ts

```diff
@@ -0,0 +1,35 @@
+/**
+ * @jest-environment node
+ */
+/* eslint-disable header/header */
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+import * as globalFlags from '../global-flags';
+import { awsuiGlobalFlagsSymbol, FlagsHolder } from '../global-flags';
+const { getGlobalFlag } = globalFlags;
+
+declare const global: typeof globalThis & FlagsHolder;
+
+afterEach(() => {
+  delete global[awsuiGlobalFlagsSymbol];
+});
+
+describe('getGlobalFlag', () => {
+  test('ensure no window in this environment', () => {
+    expect(typeof window === 'undefined').toBe(true);
+  });
+
+  test('returns undefined if there global flags are not defined', () => {
```

Comment:
typo: "there" should probably be "the"

Reply:
All these names are copy from the original tests. Updated them in both places